dashboard/templates/dashboard/index.html

```diff
@@ -3,20 +3,20 @@
 {% block content %}
     <!-- === Begin sd-latest-posts === -->
     <div id="sd-latest-posts-wrapper">
-        <section class="container">
-            <h2 class="text-center">Latest Posts</h2>
+        <div class="container">
             <div class="row">
-            <div class="col-md-12">
-                {% for post in user_posts %}
-                    <div id="post-{{ post.id }}" class="col-md-4">
-                        <div>{{ post.post_story }}</div>
-                        <div>{{ post.author.id }}</div>
-                    </div>
-                {% endfor %}
+            <!-- === Begin left-bar === -->
+            <div class="col-md-3">
+
             </div>
+            <!-- === End left-bar === -->
+            <!-- === Begin sd-posts === -->
+            {% include 'dashboard/includes/user_profile_posts.html' %}
```

Comment:
Those were left for actual content once we decide what would go there. Good catch.